mkdocs.yml

```diff
@@ -98,14 +98,16 @@ nav:
           - Formatting your ontology annotations correctly: reference/formatting-license.md
           - OBO-style term annotation: reference/go-style-annotation-property-practice.md
           - Glossary: reference/glossary.md
-          - GitHub Desktop: reference/github-desktop.md
+          -Git:
```

Comment:
don't think it matters, but space between - and Git? lol

Reply:
Thanks for catching that!